class UserNotFoundException(Exception):
    pass


class UsernameIsTakenException(Exception):
    pass
